cypher/refcard-tests/src/test/scala/org/neo4j/cypher/docgen/refcard/MathematicalFunctionsTest.scala

```diff
@@ -61,6 +61,14 @@ The absolute value.
 ###assertion=returns-one parameters=expression
 RETURN
 
+isNaN($expr)
+###
+
+Returns whether a number is `NaN`.
+
+###assertion=returns-one parameters=expression
+RETURN
+
```

Comment:
I know @martin-neotech is working on a replacement for the refcard, so leaving this comment here for visibility, this should be added to the 5.0 version of the cheat sheet